applications/laser_detection_latency/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.24)
```

Comment:
Related to this app?

Reply:
> We have a few applications which have side effects although they are not enabled.

Ah gotcha. Agreed with this fix to address the side effect, thanks!

Reply:
I'm not sure I understand the issue. What are the steps to reproduce?